src/main/java/com/leverx/learningmanagementsystem/core/security/config/CloudSecurityFilterChain.java

```diff
@@ -36,17 +37,32 @@ public SecurityFilterChain actuatorSecurityFilterChain(HttpSecurity http) throws
                 .build();
     }
 
+    /*@Bean
```

Comment:
comments must be removed, never create PR and then merge it to main with commented code (it's bad practice)